google/cloud/pubsublite/internal/wire/admin_client_impl.py

```diff
@@ -105,6 +112,29 @@ def update_subscription(
             subscription=subscription, update_mask=update_mask
         )
 
+    def seek_subscription(
+        self,
+        subscription_path: SubscriptionPath,
+        target: Union[BacklogLocation, PublishTime, EventTime],
+    ) -> Operation:
+        request = SeekSubscriptionRequest(name=str(subscription_path))
+        if isinstance(target, PublishTime):
+            ts = Timestamp()
```

Comment:
this munging is actually not necessary. The proto-plus-python library handles it.

Reply:
Done. Thanks!